### DISCUSSION TITLE
ICN5D's artwork with PyVista

### DISCUSSION THREAD

**Discussion by stla** [pyvista/pyvista#2141] [Show and tell]
**ICN5D** is the pseudo of a member of the [high.er space forum](http://hi.gher.space/forum/). He does amazing things (that I don't understand). But his animations have not a very good quality. I reproduced two of them with **PyVista**:

![](https://github.com/stla/PyVistaMiscellanous/raw/main/ICN5D_cage.gif)

![](https://github.com/stla/PyVistaMiscellanous/raw/main/ICN5D_cage2.gif)

*@ICN5D, if you see this post, I tried two times to register to the forum, but that does not work and there's no email to contact the admin.*

**Comment by stla** [comment]
Sphere-torus metamorphosis, by ICN5D.

![ICN5D_torus_to_sphere](https://user-images.githubusercontent.com/4466543/154125842-3c9c224f-179f-4b76-a747-a604b5a0a2b6.gif)

**Comment by stla** [comment]
Metamorphosis of two balls to a torus, passing by bananas. By ICN5D.

![ICN5D_two_balls_to_torus](https://user-images.githubusercontent.com/4466543/155007171-0b90a763-a7fe-4a0e-aa4b-ac033c927735.gif)

**Comment by stla** [comment]
![ICN5D_two_tori_continuous_rotation](https://user-images.githubusercontent.com/4466543/155009748-7b875ca7-7e51-4f35-876b-7ce4e6e217bd.gif)

**Comment by stla** [comment]
Wow, look at this one. Unfortunately, he doesn't provide the equation, so I can't do it with PyVista.

![FPV0AP4YMTQ6MMNX4VNHRZ3K04](https://user-images.githubusercontent.com/4466543/155011257-716c1c29-2a5d-469b-9fd0-d3ddb52e2bfb.gif)

**Comment by stla** [comment]
Incredible. I don't know why there are such "jumps" in the colors.

![ICN5D_quad_tiger_cage](https://user-images.githubusercontent.com/4466543/155013732-a7e27a53-b7ae-4f98-ab02-bc6e7451d608.gif)

**Comment by stla** [comment]
![ICN5D_topo_changes](https://user-images.githubusercontent.com/4466543/155040645-4a32a883-2dc8-4977-918a-49dfe1e7c94f.gif)

**Comment by stla** [comment]
![ICN5D_tori](https://user-images.githubusercontent.com/4466543/164196393-58e3bfca-54fa-45e2-8d1d-5a5d52aee709.gif)

**Comment by stla** [comment]
![ICN5D_tori2](https://user-images.githubusercontent.com/4466543/164196819-a18398a1-e526-4e95-979b-eba500195661.gif)

**Comment by stla** [comment]
![ICN5D_topo_changes_isoclinic](https://user-images.githubusercontent.com/4466543/179324974-16bd9998-65a8-4884-9dd1-a027c3776538.gif)

**Reply by RichardScottOZ** [reply]
Soon you will have a bowl of fruit loops.

**Comment by stla** [comment]
I started to understand this stuff. Some notes [on my blog](https://laustep.github.io/stlahblog/posts/slicedImplicitHypersurface.html).
